content/docs/guides/10-minute-tutorial.md

```diff
@@ -116,6 +123,65 @@ Open the project in IntelliJ IDEA:
 * **File -> Open... -> (Select the pom.xml)**
 * Select **Open as Project**
 
+**With Gradle**
+
+You can use the following `build.gradle` file as an example template. Just use your IDE
+to start a new Gradle-based project.
+
+```groovy
+apply plugin: 'java'
+sourceCompatibility = 11
+targetCompatibility = 11
+
+group = "hellocucumber"
+version = "1.0"
+
+// Versioning of dependencies
+wrapper.gradleVersion = '5.5.1'
+def cucumberVersion = '4.6.0'
```

Comment:
Please use the Hugo version tags so this version is made dynamic

https://github.com/cucumber/docs.cucumber.io/blob/fc79bdf0fe30968d2cd2f44c43b410d589fe0ae7/content/docs/installation/java.md#L24